Core/src/main/java/com/plotsquared/core/player/PlotPlayer.java

```diff
@@ -866,9 +867,9 @@ public void sendTitle(
             final int fadeIn, final int stay, final int fadeOut,
             final @NonNull Template... replacements
     ) {
-        final Component titleComponent = MiniMessage.get().parse(title.getComponent(this), replacements);
+        final Component titleComponent = MiniMessage.miniMessage().deserialize(title.getComponent(this), TemplateResolver.templates(replacements));
         final Component subtitleComponent =
-                MiniMessage.get().parse(subtitle.getComponent(this), replacements);
+                MiniMessage.miniMessage().deserialize(subtitle.getComponent(this), TemplateResolver.templates(replacements));
```

Comment:
Should we not be statically storing a MiniMessage here?

Reply:
4.1.0 -> 4.2.0 deprecates naming schemes e.g. the getter in MiniMessage and replaces it with #miniMessage. The TemplateResolver was already used under the hood in #parse, same like for #miniMessage, it does the same like #get. This is a chore PR that addresses the deprecation scheme, I didn't modify public API, more changes to MM is coming soon tm.

Reply:
? I mean adding a `private static final MiniMessage MINI_MESSAGE = MiniMessage.miniMessage();`

Reply:
Yeah, that was solely for clarification.